### ISSUE TITLE
[24] Update Market Place Entry 

### ISSUE THREAD

**Issue by mpalat** [eclipse-jdt/eclipse.jdt.core#3012]
Ref the [similar issue](https://github.com/eclipse-jdt/eclipse.jdt.core/issues/2962) in the previous release

**Comment by jarthana** [comment]
I have created a marketplace entry based on https://download.eclipse.org/jdt/updates/4.35-P-builds/
This is still under moderation. Will update/close once it is live.

**Comment by mpalat** [comment]
Available - closing

**Comment by stephan-herrmann** [comment]
The marketplace entry links to the composite repo https://download.eclipse.org/jdt/updates/4.35-P-builds/ 
Is this a deliberate decision?
* It _might_ allow us to provide hotfix updates in the the future
   * this would still require a functional Y-build, which I'm not sure how long this can be kept.
* It requires full scrutiny before ever creating another P-build (which immediately contributes to the composite)
* If people select to see all versions they could still attempt to install an outdated P-build

If, however, we'd change the address to the explicit release from yesterday, then all flexibilities as well as related risks would be shut off.

If the consequences have been considered, I'm fine. Just wanted to make this explicit.

**Comment by jarthana** [comment]
> The marketplace entry links to the composite repo https://download.eclipse.org/jdt/updates/4.35-P-builds/ Is this a deliberate decision?
> 

Yes, it has always been like this. In the past, creating P build and adding to the composite were two different steps sandwiching a manual testing. In the past, especially after we moved to the quarterly eclipse release, we rarely had subsequent builds after GA. So, we are mostly safe.

May be for future, if we can pass an argument to the jdt-patch-feature that will let us skip/do the composite thing, that'll be nice.
